### ISSUE TITLE
[travis] Builds failing due to Selenium Chrome Driver

### ISSUE THREAD

**Issue by alfonso-presa** [Polymer/tools#3353]
Somehow Chrome Driver is failing when trying to start (travis is providing chrome 62 and driver is only compatible with >69). See [stack trace here](https://travis-ci.org/Polymer/tools/jobs/491633759#L684):

```
@polymer/esm-amd-loader: chrome 62                Tests failed: {"value":
{"stacktrace":"org.openqa.selenium.SessionNotCreatedException: session not created: Chrome version must be >= 69.0.3497.0\n  (Driver info: chromedriver=2.43.600233 
(523efee95e3d68b8719b3a1c83051aa63aa6b10d),platform=Linux 4.4.0-101-generic x86_64) 
(WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 60.15 seconds\nBuild info: version: '3.12.0', revision: '7c6e0b3', time: '2018-05-08T15:15:08.936Z'\nSystem info: host: 'travis-job-1d5a7da7-f06e-4ac0-b8dc-1014bac8a99d', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-101-generic', java.version: 
'1.8.0_151'\nDriver info: driver.version: unknown\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\tat java.lang.reflect.Constructor.newInstance(Constructor.java:423)\n\tat 
```
-------
Edit: old content
I made a [PR](https://github.com/Polymer/tools/pull/3352) and the build failed. Digging a little bit inside the issue I noticed Saucelabs is being used and, very wisely, it's credentials are encrypted, which makes travis make them not available during  [unsafe PRs builds](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml).

I would expect that, as having a continuously failing PR check condition removes automatically any value that condition could provide, unsafe PR tests should not be executed against saucelabs, so that the build color provides quick feedback on the quality of the code behind the PR (and we external contributors don't get crazy about how to fix the build :-) ).

**Comment by TimvdLippe** [comment]
We run Sauce on Travis push. The usage of encrypted variables is therefore not related to Travis PR builds. I have restarted the failing job, as it was failing on a Selenium error.

**Comment by alfonso-presa** [comment]
Quick response! Thanks! 

I did relaunch the job myself (via force push), and it did fail (and it looks like the relaunching you did, did fail also). I thought it was somehow related with the `WCT_SAUCE=$TRAVIS_SECURE_ENV_VARS SKIP_LOCAL_BROWSERS=true` env variable and the (last entry in the travis.yml)[https://github.com/Polymer/tools/blob/master/.travis.yml#L56], but according to your comments, it does not ^_^. Sorry!

**Comment by alfonso-presa** [comment]
@TimvdLippe Sorry to bother again.... But the more I look at it, the more it looks like the `WCT_SAUCE` variable is not being set in the (last matrix job)[https://github.com/Polymer/tools/blob/master/.travis.yml#L56].... And it looks like that job is meant to be used against saucelabs.

I know how to fix this (basically excluding the last job from PRs)... do you want me to PR with the fix?

**Comment by TimvdLippe** [comment]
Your test fails on the Selenium Chrome driver and is unrelated to Sauce. I think we need to update our `run-wct.sh` scripts to also skip on `SKIP_LOCAL_BROWSERS`. CC @aomarks

**Comment by alfonso-presa** [comment]
Ok. I've reopened and reformulated the issue.

**Comment by alfonso-presa** [comment]
@TimvdLippe I think I know understand how this is supposed to work. Wouldn't skipping also in 
`SKIP_LOCAL_BROWSERS` fail as chrome and firefox addons are not avail in the job? I think it's now failing because without chrome addon, travis is providing an older (~62) chrome version.

**Comment by alfonso-presa** [comment]
Looks like the fix in my PR worked :-). But of course is up to you to decide the best implementation. I was just curious about it and making the PR did help me to understand your point. Thanks!

**Comment by alfonso-presa** [comment]
I lack a lot of context to be able to propose a consistent solution to this issue by myself :-).

Seems like you've got it clean & clear, but I'm going to sum up what I've learned so far, just in case it helps.

I understand form @TimvdLippe comments the following from the (5th job triggered in travis builds)[https://github.com/Polymer/tools/blob/master/.travis.yml#L56]:
* PR builds will not be executed against Saucelabs browsers (as not credentials are in place, and should not be)
* PR builds should not be executed with the `SKIP_LOCAL_BROWSERS=true` and at the same time use addons to provide local browsers (and it truly doesn't have any sense in terms of semantics of the variable name).

I've also learned by trial & error that running the 5th job:
* without chrome and firefox addons cause failures, because it tries to use browsers (even though it doesn't have saucelabs, and it has the `SKIP_LOCAL_BROWSERS=true`) and it uses the chrome (and probably firefox) set in place by travis, and they are older versions.
* with chrome and firefox addons it doesn't work unless you specify `SKIP_LOCAL_BROWSERS=true` env variable. Having the 3 of them (the two addons and the variable) in place makes the build pass. 

If all this is true, it raises some questions on my mind:
- What are the integration tests expected to be executed in the 5th job without local browsers and without a cloud browser provider?
- Does it has sense to directly disable that 5th job for PRs? May be it doesn't have sense because it will also be disabled for internal PRs (and not only untrusted).

From what I understand, adding an else if in (this line)[https://github.com/Polymer/tools/blob/master/packages/esm-amd-loader/test/run-wct.sh#L5] (and every other run-wct.sh script) with a `SKIP_LOCAL_BROWSERS != 'true'` check, is the way to go. Right?
